components/pages/library/BookDetails.js

```diff
@@ -149,7 +150,7 @@ export default function BookDetails({ serverBook, serverContent }) {
 
     return (
       <>
-        <h3>Summary:</h3>
+        <h1>Summary</h1>
```

Comment:
There are cases when both the `renderQuotes` and `renderStories` function are used to render their return values; both functions return an `h1`. Please avoid this because every page should only have one `h1` heading.

Reply:
here the context of the whole component is missing, that makes quite sense